### PR TITLE
feat: update Kubernetes to v1.26.0-alpha.1

### PR DESCRIPTION
**Pull request by smira**
Talos 1.3.0 will ship with Kubernetes 1.26.0.

See https://github.com/kubernetes/kubernetes/releases/tag/v1.26.0-alpha.1

Signed-off-by: Andrey Smirnov <andrey.smirnov@talos-systems.com>

### CODE REVIEW

File: pkg/cluster/kubernetes/talos_managed.go [hunk 1]
```diff
@@ -69,6 +69,11 @@ var deprecations = map[string][]string{
 		"podsecuritypolicies.v1beta1.policy",
 		"runtimeclasses.v1beta1.node.k8s.io",
 	},
+	"1.25->1.26": {
```
Comment (frezbo) [inline pkg/cluster/kubernetes/talos_managed.go:72]: never heard of these apis at all :sweat_smile:

File: website/content/v1.3/_index.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ linkTitle: "Documentation"
 cascade:
   type: docs
 lastRelease: v1.3.0-alpha.0
-kubernetesRelease: "1.25.1"
+kubernetesRelease: "1.26.0-alpha.1"
 prevKubernetesRelease: "1.25.0"
```
Comment (frezbo) [inline website/content/v1.3/_index.md:9]:
```suggestion
prevKubernetesRelease: "1.25.1"
```

File: website/content/v1.3/_index.md [hunk 3]
```diff
@@ -5,7 +5,7 @@ linkTitle: "Documentation"
 cascade:
   type: docs
 lastRelease: v1.3.0-alpha.0
```
Comment (frezbo) [inline website/content/v1.3/_index.md:7]: we don't have alpha.0 yet, right?

File: website/content/v1.3/_index.md (hunk 2 above)
Comment (smira) [reply website/content/v1.3/_index.md]: fixed

**Pull request merged**